component/common/application/matter/example/bridge_dm/matter_drivers.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "matter_events.h"
+#include <lwip/ip_addr.h>
+#include <platform/CHIPDeviceLayer.h>
+
+CHIP_ERROR matter_driver_bridge_light_init(void);
+void matter_driver_bridge_endpoint_assign(void);
+void matter_driver_bridge_setup_server(void);
+void matter_driver_bridge_recv_server(void);
```

Comment:
The src code for setup_server and recv_server is not found

Reply:
I think a better idea is to create a class for bridge, then implement callbacks which will be called when an endpoint joins or leaves.